kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/io/kotest/matchers/collections/DescribePartialMatchesInCollectionTest.kt

```diff
@@ -0,0 +1,52 @@
+package io.kotest.matchers.collections
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.string.shouldContainInOrder
+
+class DescribePartialMatchesInCollectionTest: WordSpec() {
```

Comment:
I liked this test.
It helps understand wtf `describePartialMatchesInCollection` does